deploy/distrib/src/main/resources/README.TXT

```diff
@@ -56,12 +58,16 @@ Contents of this package:
 |   |   `-- lib
 |   |       `-- *.jar                    				Common libs that can be shared by several webapps (H2 listener)
 |   `-- welcome-content
-|       `-- favicon.ico
-|    
+|   |   `-- favicon.ico
+|   `-- bonita-start(.sh|.bat)                          Script used to start Bonita BPM Tomcat bundle
+|
 |-- platform-setup							            Platform creation/initialization files (Database scripts + configuration files)
```

Comment:
my previous comment about naming this "setup" as in other distrib is still valid